nhost/nhost.go

```diff
@@ -293,14 +293,14 @@ func Servers() ([]Server, error) {
 	return response, err
 }
 
-//  generates fresh config.yaml for /nhost dir
+// generates fresh config.yaml for /nhost dir
 func GenerateConfig(options App) Configuration {
 
 	log.Debug("Generating app configuration")
 
 	hasura := Service{
 		Environment: map[string]interface{}{
-			"hasura_graphql_enable_remote_schema_permissions": false,
+			"hasura_graphql_enable_remote_schema_permissions": true,
```

Comment:
isn't this supposed to be upper case?

Reply:
Shouldn't be a problem, upcases here https://github.com/nhost/cli/blob/main/nhost/utils.go#L26